.github/workflows/main.yml

```diff
@@ -57,6 +57,9 @@ jobs:
           - stable
     steps:
       - uses: actions/checkout@v3
+        with:
+          submodules: true
+      - run: git submodule init && git submodule update --remote --recursive
```

Comment:
If you look at the last build run, `submodules: true` already initializes and fetches the submodule:

https://github.com/freespek/solarkraft/actions/runs/9465023037/job/26073622926?pr=100#step:2:422

```suggestion
```